cmake/macros/macro_deal_ii_invoke_autopilot.cmake

```diff
@@ -148,6 +148,11 @@ MACRO(DEAL_II_INVOKE_AUTOPILOT)
   # Only mention release and debug targets if it is actually possible to
   # switch between them:
   IF(${DEAL_II_BUILD_TYPE} MATCHES "DebugRelease")
+
+    SET(CMAKE_BUILD_TYPE "Debug" CACHE STRING
+    "Choose the type of build, options are: Debug, Release"
+    FORCE)
```

Comment:
One last change :-)

Can you guard the ```SET``` with ```IF("${CMAKE_BUILD_TYPE}" STREQUAL "DebugRelease")``` instead and relocate it to the beginning of this file?